src/pre_process/m_initial_condition.fpp

```diff
@@ -283,6 +285,1204 @@ contains
         end do
 
     end subroutine s_perturb_surrounding_flow ! ----------------------------
+	
+	subroutine s_superposition_instability_wave() ! ------------------------------
```

Comment:
unclear what this subroutine is doing. please remove the `write(*,*)` statement and put a doxygen docstring above the subroutine call (line 288) like the other subroutines have.